tests/python/PEP8NotebookBearTest.py

```diff
@@ -22,6 +22,19 @@
    "source": [
     "a markdown cell is not a code cell"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [
+    "'x =' + ' 1 +' * 100 + ' 1'"
+   ],
+   "source": [
+    "max_line_length = 0"
```

Comment:
this 'source' doesnt match the 'outputs' above, making this an invalid file.

however more importantly, this does not test your code changes.

your changes to the tests need to *prove* that your code changes work correctly.  See examples that other people have given.